missing information about API authorisation usages (#4284)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/api/rest-api-v1.md

## Permissions

> Users can be authorized to perform API calls by configuring the corresponding fields located in the [**Centreon Authentication**](../monitoring/basic-objects/contacts-create.md#onglet-authentification-centreon) tab.

Reviewer: path to the page

- The [**Configuration API**](#configuration) can only be called by administrators.
- The [**Realtime API**](#realtime-information) can be called by non-administrator users as long as the corresponding field is checked.

> - Administrators are able to call both the [**Configuration API**](#configuration) and the [**Realtime API**](#realtime-information) regardless of the corresponding field of the API being checked. They are also the only ones allowed to use [**CLAPI**](clapi.md) while others can only use Rest API.

Reviewer: ```suggestion
- Administrators are able to call both the [**Configuration API**](#configuration) and the [**Realtime API**](#realtime-information) even if these fields are not checked. They are also the only ones allowed to use [**CLAPI**](clapi.md) while others can only use the Rest API.
```

## Permissions

> Users can be authorized to perform API calls by configuring the corresponding fields located in the **Configuration > Users > Contacts/Users >** [**Centreon Authentication tab**](../monitoring/basic-objects/contacts-create.md#centreon-authentication).

Reviewer: ```suggestion
Users can be authorized to perform API calls by configuring the corresponding fields located in the **Configuration > Users > Contacts/Users >** page, on the [**Centreon Authentication** tab](../monitoring/basic-objects/contacts-create.md#centreon-authentication).
```
